aep/general/0158/aep.md.j2

```diff
@@ -148,10 +148,30 @@ used. It is not necessary to document this behavior.
 **Note:** While a reasonable time may vary between APIs, a good rule of thumb
 is three days.
 
+### Freshness
+
+Since a request with the same filter and `page_token` **should** return the
+same results unless the page token has expired, items returned **may** have
+been deleted or modified since the initial request, and paging through the list
+**may** not include all current items.
+
+If the service is capable of detecting underlying changes in the listing since
```

Comment:
```suggestion
If the service is capable of detecting underlying changes in the list of results since
```

Not sure about the term "the listing" in this context...sounds to me like a marketplace term.